### PR TITLE
Adding french translations to the training form

### PR DESCRIPTION
**Pull request by sylviamclaughlin**
# Summary | Résumé

Adding French translations to all strings related to the training form.

### CODE REVIEW

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -482,25 +489,27 @@ msgid ""
 "If Saas is to be used for all of CDS, select FC 110804. If the Saas is to be "
 "used specifically for your BU, please select the appropriate FC."
 msgstr ""
-"Si ce SaaS doit être utilisé par tous le personnel du SNC, sélectionnez FC 110804. Si le"
-"Saas doit être utiliséutilisé spécifiquement pour votre BU, veuillez "
-"sélectionner le FC approprié."
+"Si ce SaaS doit être utilisé par tous le personnel du SNC, sélectionnez FC "
+"110804. Si leSaas doit être utiliséutilisé spécifiquement pour votre BU, "
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:493]: There should be a space between `le` and `SaaS`
And we should remove one of the two `utilisé`

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -816,78 +833,112 @@ msgstr "Bienvenue"
...
+msgid "Submit a Training Request Form"
+msgstr "Soumettre un formulaire de demande de formation"
+
+#: templates/index.html:51
+msgid ""
+"Please fill out this form if you would like to request training or courses "
+"related to your job."
+msgstr ""
+"Veuillez remplir ce formulaire si vous souhaitez demander une formation ou "
+"des courslié à votre travail."
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:882]: We should add a space between `cours` and `lié`

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1079,6 +1135,11 @@ msgstr ""
 msgid "The request has been upated and the approver will be notified."
 msgstr "La demande a été mise à jour et l'approbateur en sera informé."
 
+#: templates/request/saas_request.html:11
+#: templates/request/view_request.html:11
+msgid "Software as a Service Request Form"
+msgstr "Logiciel en tant que formulaire de demande de service"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1141]: I think the syntax got scrambled:
"Logiciel en tant que service" should be together.

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
...
 
+#: templates/training/training_request.html:8
+#: templates/training/view_request.html:8
+msgid "Training Form Request"
+msgstr "Demande de formulaire de formation"
+
+#: templates/training/training_request.html:11
+#: templates/training/view_request.html:11
+msgid "Training Request Form"
+msgstr "Demande de formulaire de formation"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1249]: The order probably got scrambled:

It's a "Formulaire de demande de formation". Current syntax would mean: Request for a training form 😄

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
...
+
+#: templates/training/training_request.html:12
+msgid "Please fill out this form to apply for training."
+msgstr "Veuillez remplir ce formulaire pour postuler à une formation."
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1253]: To be confirmed: postuler probably needs to be modified by another verb as this is mostly associated to applying to a job, in my experience.

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
...
+"your training form."
+msgstr ""
+"Votre responsable a reçu une notification du formulaire et va l'examiner. Si "
+"approuvé, le formulaire sera envoyé à l'approbateur S32 pour être signé et "
+"finalisé. Tout au long du processus, vous recevrez des notifications sur "
+"l'avancement de votre formulaire de formation."
+
+#: templates/training/training_thanks.html:17
+msgid "Download completed form"
+msgstr ""
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1273]: "Télécharger le formulaire complété"

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
...
+msgid "Submitted Training Requests"
+msgstr "Demandes de formation soumises"
+
+#: templates/training/view_all_requests.html:10
+msgid ""
+"This is the list of training requests that you have submitted. Click on the "
+"request name to view more information."
+msgstr ""
+"Voici la liste des demandes de formation que vous avez soumises. Cliquez sur "
+"le bouton demander un nom pour afficher plus d'informations."
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1285]: I think the meaning in French doesn't match the English

File: saas_app/locale/fr/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -1173,10 +1238,136 @@ msgstr "Résultats de recherche"
...
 msgid "Last Updated:"
 msgstr "Dernière mise à jour:"
 
+#: training_request/forms.py:23
+msgid "Position Title"
+msgstr "Titre du poste"
+
+#: training_request/forms.py:24
+msgid "E-Mail"
+msgstr "E-Mail"
```
Comment (gcharest) [inline saas_app/locale/fr/LC_MESSAGES/django.po:1305]: Courriel

File: saas_app/locale/fr/LC_MESSAGES/django.po (hunk 5 above)
Comment (sylviamclaughlin) [reply saas_app/locale/fr/LC_MESSAGES/django.po:1253]: Who do you think is the best person/resource to confirm with? Translations?

And BTW, thank you so much for going over the translations!

**Pull request merged**